test/sanity/run-test.sh

```diff
@@ -36,6 +38,15 @@ function install_csi_sanity_bin {
   popd
 }
 
+function provision_samba_server {
+  echo 'Installing smaba server on localhost'
+  apt-get update -y
+  apt-get install -y samba
```

Comment:
I think you need to remove L43-44 since the error said:
```
Digest: sha256:8dc1c23455f970c1275be0cdf1be35c9f5a7d1d2ddd5860b6b25a9cda5e4403e
Status: Downloaded newer image for dperson/samba:latest
50e41ff68f92acbec1891a54fb564b7088187a618f08215b47d7cb1f9c8bd0b1
docker: Error response from daemon: driver failed programming external connectivity on endpoint samba (ceaaefc07a661bc2856a41178f224256195277d8e4b7e48c5cc689de0c333755): Error starting userland proxy: listen tcp 0.0.0.0:445: bind: address already in use.
pkill -f smbplugin
Makefile:75: recipe for target 'sanity-test' failed
make: *** [sanity-test] Error 1
Error: Process completed with exit code 2.
```

install `apt-get install -y samba` would already start a samba server.